RetrospectiveExtension.Frontend/components/feedbackBoardContainer.tsx

```diff
@@ -705,7 +705,7 @@ class FeedbackBoardContainer extends React.Component<FeedbackBoardContainerProps
     }
   }
 
-  private clickWorkflowStateCallback = (clickedElement: React.MouseEvent<HTMLElement>, newPhase: WorkflowPhase) => {
+  private clickWorkflowStateCallback = (clickedElement: React.MouseEvent<HTMLElement> | React.KeyboardEvent<HTMLDivElement>, newPhase: WorkflowPhase) => {
```

Comment:
In the props for the `WorkflowStage`, the definition for the `clickEventCallback` this is passed to is defined as so:
```
clickEventCallback: (clickedElement: React.MouseEvent<HTMLElement> | React.KeyboardEvent<HTMLDivElement>, newPhase: WorkflowPhase) => void;
```

without adding this or, JEST throws errors that there is a type assignment mismatchment. The other option to resolve this issue would be to separate `clickEventCallback` into two separate calls, however I think that would significantly complicate the `WorkflowStage` component which is used by components other than this one.

Reply:
Yes! This gets around the annoying error that I was seeing on the feedback board container